netket/graph/lattice.py

```diff
@@ -261,6 +177,17 @@ def __init__(
                 to :math:`k`-nearest neighbor sites (measured by their Euclidean distance)
                 are included in the graph. The edges can be distiguished by their color,
                 which is set to :math:`k - 1` (so nearest-neighbor edges have color 0).
+                By default, nearest neighbours (:code:`max_neighbor_order=1`) are autogenerated
+                unless :code:`custom_edges` is passed.
+            custom_edges: Lists all edges starting in one unit cell, which are repeated
+                in every unit cell of the constructed lattice.
+                Should be a list of tuples; each tuple should contain the following:
+                    index of the starting point in the unit cell
+                    index of the endpoint in the unit cell
+                    vector distance between the two
+                    color of the edge (optional)
+                If colors are not supplied, they are assigned sequentially starting from 0.
+                Cannot be used together with `max_neighbour_order`.
```

Comment:
```suggestion
                Cannot be used together with `max_neighbor_order`.
```
I don't mind British spelling in our docs, but at least for the arguments we need to stick to the spelling in the code. :grin:

Reply:
Oops :smile: